R-package/R/lgb.Booster.R

```diff
@@ -930,13 +940,30 @@ lgb.get.eval.result <- function(booster, data_name, eval_name, iters = NULL, is_
     stop("lgb.get.eval.result: data_name and eval_name should be characters")
   }
 
-  # Check if recorded evaluation is existing
-  if (is.null(booster$record_evals[[data_name]])) {
-    stop("lgb.get.eval.result: wrong data name")
+  # NOTE: "start_iter" exists in booster$record_evals but is not a valid data_name
+  data_names <- setdiff(names(booster$record_evals), "start_iter")
+  if (!(data_name %in% data_names)) {
+    stop(paste0(
+      "lgb.get.eval.result: data_name "
+      , shQuote(data_name)
+      , " not found. Only the followwing datasets exist in record evals: ["
+      , paste(data_names, collapse = ",")
```

Comment:
```suggestion
      , paste(data_names, collapse = ", ")
```
I think it will be more readable.

Reply:
sure